phpstan.neon

```diff
@@ -9,8 +9,8 @@ parameters:
     checkMissingIterableValueType: false
     reportUnmatchedIgnoredErrors: false
 
-    excludes_analyse:
-        - %currentWorkingDirectory%/src/Bundle/Controller/ControllerTrait.php
+    excludePaths:
+        - %currentWorkingDirectory%/src/Bundle/Controller/*
```

Comment:
👍 for now, but it would be good to come back to this change after the merge and figure out how to exclude as few classes as possible